From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Tue, 21 Mar 2023 04:00:18 +0100
Subject: panels/mouse: Do not bind sensitivity to touch pad settings

If doing so, these options may be marked as sensitive by GSettingsBind,
ignoring the send-events global setting binding.
---
 panels/mouse/cc-mouse-panel.c | 15 ++++++++++-----
 1 file changed, 10 insertions(+), 5 deletions(-)

diff --git a/panels/mouse/cc-mouse-panel.c b/panels/mouse/cc-mouse-panel.c
index 3d3a317..ae5fb69 100644
--- a/panels/mouse/cc-mouse-panel.c
+++ b/panels/mouse/cc-mouse-panel.c
@@ -333,23 +333,28 @@ setup_dialog (CcMousePanel *self)
 
   g_settings_bind (self->touchpad_settings, "natural-scroll",
                    self->touchpad_scroll_direction_row, "use-default",
-                   G_SETTINGS_BIND_INVERT_BOOLEAN);
+                   G_SETTINGS_BIND_INVERT_BOOLEAN |
+                   G_SETTINGS_BIND_NO_SENSITIVITY);
 
   g_settings_bind (self->touchpad_settings, "speed",
                    gtk_range_get_adjustment (GTK_RANGE (self->touchpad_speed_scale)), "value",
-                   G_SETTINGS_BIND_DEFAULT);
+                   G_SETTINGS_BIND_DEFAULT |
+                   G_SETTINGS_BIND_NO_SENSITIVITY);
 
   g_settings_bind (self->touchpad_settings, "tap-to-click",
                    self->tap_to_click_switch, "active",
-                   G_SETTINGS_BIND_DEFAULT);
+                   G_SETTINGS_BIND_DEFAULT |
+                   G_SETTINGS_BIND_NO_SENSITIVITY);
 
   g_settings_bind (self->touchpad_settings, "two-finger-scrolling-enabled",
                    self->touchpad_scroll_method_row, "use-default",
-                   G_SETTINGS_BIND_DEFAULT);
+                   G_SETTINGS_BIND_DEFAULT |
+                   G_SETTINGS_BIND_NO_SENSITIVITY);
 
   g_settings_bind (self->touchpad_settings, "edge-scrolling-enabled",
                    self->touchpad_scroll_method_row, "use-default",
-                   G_SETTINGS_BIND_INVERT_BOOLEAN);
+                   G_SETTINGS_BIND_INVERT_BOOLEAN |
+                   G_SETTINGS_BIND_NO_SENSITIVITY);
 
   setup_touchpad_options (self);
 
